data/bucket_summary.py

```diff
@@ -0,0 +1,81 @@
+from influxdb_client import InfluxDBClient
+import pandas as pd
+import warnings
+from influxdb_client.client.warnings import MissingPivotFunction
+
+warnings.simplefilter("ignore", MissingPivotFunction)
+
+client = InfluxDBClient(
+    url="http://161.97.156.125:8086",
+    token="JCDOYKFbiC13zdgbTQROpyvB69oaUWvO4pRw_c3AEYhTjU998E_X_oIJJOVAW24nAE0WYxMwIgdFSLZg8aeV-g==",
+    org="distributed-training"
+)
```

Comment:
Can we move the token to be in a non-public .env file to prepare for different levels of access to our InfluxDB buckets?